src/views/Resources.vue

```diff
@@ -2,7 +2,7 @@
   <main>
     <form class="search-form">
       <Input :value.sync="searchTerm" name="Search" placeholder="Search resources..." />
-      <Select :value.sync="selectedTypes" name="Resource Type">
+      <Select :value.sync="selectedType" name="Resource Type">
```

Comment:
if I remember correctly our plan was to use a multiselect there in the nearer future and that was the reason why we implemented it for an array of multiple types. Do you think we will keep a single-select?

Reply:
I saw errors in the browser console, which is why I changed it. Anyway, we should aim for the multi-select when rewriting the select component. For now I prefer not having errors in my browser console :P.